html/changelogs/example.yml

```diff
@@ -1,37 +0,0 @@
-################################
```

Comment:
You've deleted this file. Please, restore it.

Reply:
Done.